codeworld-compiler/codeworld-compiler.cabal

```diff
@@ -54,16 +54,18 @@ Library
     directory,
     exceptions,
     filepath,
-    ghc == 8.6.5,
-    ghc-boot-th == 8.6.5,
+    ghc == 8.10.*,
+    ghc-boot,
+    ghc-boot-th == 8.10.*,
+    ghc-lib-parser,
```

Comment:
I don't see where this is used.

If it's needed, ghc-lib-parser changes dramatically between versions (it's just a copy of parts of ghc), so it needs version bounds.  But it would be better to leave it out and update the ghc code, if it's not too difficult.